support/reftest/data.yml

```diff
@@ -28,10 +28,6 @@ test_data:
 - path: /origin/rpms/bash/4.4.19/8.el8_0/fd431d51/bash-4.4.19-8.el8_0.x86_64.rpm
   sha256: 4164ff2c0116d666578ba5e456ab03b88788dfb42fb93fc91b2c1da709d86686
   content-type: application/x-rpm
-# listing
```

Comment:
I think we should not remove this but instead prevent it from being uploaded.

I see the reftest/data.yml as serving two purposes:

1. Declares the set of data which can be copied from legacy CDN to populate exodus CDN for testing.

2. Declares a set of endpoints which, given a healthy CDN, should be possible to query and observe the specified properties.

Since we don't upload listing files by traditional means any more then indeed listing isn't valid for (1) now.  But it's still valid for (2), isn't it?  i.e. even with the new method of handling listings, it's still expected that I should be able to query /content/dist/rhel/server/5/5.7/listing and get some content of sha256sum b51f4ddc06fddec9e73892671f1f25300ccd4235fa798afd01caf96446dc2bf1 ?

In that case I don't think we should purge it from data, because it's a valuable test case.  (I thought we already had a test of the form "iterate through all these entries, query the path, verify we got a 200 response, verify checksum if defined" - and if we don't have that we probably should.)

Could we instead do something like `deploy: false` and then update support/reftest/reftest to skip any item where deploy is false?  So it could continue to serve both the above use-cases.